api4/cloud.go

```diff
@@ -630,23 +629,6 @@ func handleCWSWebhook(c *Context, w http.ResponseWriter, r *http.Request) {
 			c.Err = model.NewAppError("SendCloudWelcomeEmail", "api.user.send_cloud_welcome_email.error", nil, err.Error(), http.StatusInternalServerError)
 			return
 		}
-	case model.EventTypeSubscriptionChanged:
```

Comment:
@neallred We're good to completely get rid of this too then?

Reply:
Yep, given we do the tasks from HandleLicenseChange.